enforcer/datapath/datapath_tcp.go

```diff
@@ -505,21 +476,21 @@ func (d *Datapath) processNetworkSynPacket(context *pucontext.PUContext, conn *c
 	// we must drop the connection and we drop the Syn packet. The source will
 	// retry but we have no state to maintain here.
 	if err != nil || claims == nil {
-		d.reportRejectedFlow(tcpPacket, conn, collector.DefaultEndPoint, context.ManagementID, context, collector.InvalidToken, nil)
-		return nil, nil, fmt.Errorf("syn packet dropped because of invalid token: %s", err)
+		d.reportRejectedFlow(tcpPacket, conn, collector.DefaultEndPoint, context.ManagementID(), context, collector.InvalidToken, nil)
+		return nil, nil, fmt.Errorf("Syn packet dropped because of invalid token %v %+v", err, claims)
 	}
 
 	txLabel, ok := claims.T.Get(enforcerconstants.TransmitterLabel)
 	if err := tcpPacket.CheckTCPAuthenticationOption(enforcerconstants.TCPAuthenticationOptionBaseLen); !ok || err != nil {
-		d.reportRejectedFlow(tcpPacket, conn, txLabel, context.ManagementID, context, collector.InvalidFormat, nil)
-		return nil, nil, fmt.Errorf("tcp authentication option not found: %s", err)
+		d.reportRejectedFlow(tcpPacket, conn, txLabel, context.ManagementID(), context, collector.InvalidFormat, nil)
+		return nil, nil, fmt.Errorf("TCP Authentication Option not found %v", err)
```

Comment:
use "TCP authenticating option not found: %s"

1) colon for consistency concatenating errors passed from a function call 
2) %s works and using %v does just another reflect. Also, just clean up %v's in the PR wherever this PR is adding them.